src/components/Navbar.jsx

```diff
@@ -1,27 +1,38 @@
-import React, { useState } from 'react';
+import React, {  useState } from 'react';
+import { Link, useLocation} from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import { GiHamburgerMenu } from 'react-icons/gi';
+import { BiDownArrow } from 'react-icons/bi';
 import { AiOutlineClose } from 'react-icons/ai';
-import { Link, NavLink, useLocation } from 'react-router-dom';
 import { searchlog, techLogW } from '../assets';
 import Button from './Button';
+import checkIsLogged from '../utils/isLoggedin';
+import logOut  from '../utils/logOut';
+
 
 const Navbar = () => {
   const { pathname } = useLocation();
   const [close, setClose] = useState(false);
+  const {token}=useSelector((state)=>{return state.auth})
+  console.log(token)
   if (
     pathname === '/login' ||
     pathname === '/signup' ||
-    pathname === '/dashboard/users'||
-    pathname.startsWith('/signup/')
```

Comment:
Can you keep this line `pathname.startsWith('/signup/')` we need it because the navbar will appear again if we pass some values in the link params 

- Eg: in google authentication if the user isn't already registered in the database he will be redirected with a link like this:`https://e-comm-team.../api/users/signup/?email=charles%40gmail%23com&?name..` . In this case the nav bar comes back in the signup even if we don't need it.